src/modules/sdai/LiveSDAIBalance.tsx

```diff
@@ -84,3 +78,24 @@ function useRefresh(ms: number) {
     };
   }, []);
 }
+
+const getVisibleDecimals = (daiBalance: BigNumber | undefined): number => {
+  const changeRatio = 1.55e-9;
+  const value = daiBalance?.multipliedBy(changeRatio);
+
+  if (!value) {
+    return 0;
+  }
+
+  const e = value.e;
+
+  if (!e || e >= 0) return 0;
+
+  const decimals = -e;
+
+  if (decimals === 1) return 2;
```

Comment:
This could be written as 1 liner but I am not sure if it makes it more redable 🤔 

`Math.min(Math.max(2, decimals), 7)`

Reply:
imho ifs looks easier to read here, but up to you